build-support/bin/changelog.py

```diff
@@ -9,6 +9,10 @@
 from textwrap import dedent
 from typing import List
 
+from common import die
```

Comment:
Would a relative import be better here?

Reply:
Perhaps, but we've always done it this way for other scripts.